### PR TITLE
Add Getting Start Page

### LINKED ISSUE #18: feat: add getting started page
we would like to add a new page to show the getting started page to share details on how to use the sample plugin and app to get head started.:

Do This:
- Add the button `Getting Started` on the home page hero 
- Add a component for the page that displays data from [markdown](https://gist.github.com/ajitbohra/9b0a84366e5dc8ab84ab5da809d6db08) using the [react-markdown](https://github.com/remarkjs/react-markdown ) library

**Resources:**
https://github.com/remarkjs/react-markdown 
https://retool.com/blog/react-markdown-component-the-easy-way-to-create-rich-text 
https://nextjs.org/learn-pages-router/basics/dynamic-routes/render-markdown 
https://blog.logrocket.com/how-to-safely-render-markdown-using-react-markdown/ 
https://stackoverflow.com/questions/42928530/how-do-i-load-a-markdown-file-into-a-react-component  
https://developer.wordpress.org/news/2023/04/25/how-webpack-and-wordpress-packages-interact/ 
https://samhermes.com/posts/customize-default-wp-scripts-config/

### PR DESCRIPTION
**Pull request by deepakgithub11**
Closes #18 
added getting started button page
added md file also applied some CSS on file

### CODE REVIEW

File: src/components/gettingstarted.js
Comment (ajitbohra) [inline src/components/gettingstarted.js:1]: this should got under `pages`

File: src/components/gettingstarted.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import ReactMarkdown from "react-markdown";
+import Gs from "./gs.md"
+import Markdown  from "react-markdown";
+import remarkGfm from 'remark-gfm';
+import "./md.css"
```
Comment (ajitbohra) [inline src/components/gettingstarted.js:5]: organize imports as external, WordPress and Internal

File: src/components/gs.md
Comment (ajitbohra) [inline src/components/gs.md:1]: this should go under `docs` folder all md files in one location

File: src/components/index.js [hunk 2]
```diff
@@ -7,3 +7,4 @@ export { default as Banner } from './banner';
 export { default as Logo } from './logo';
 export { default as ContentLoader } from './content-loader';
 export { default as ScrollToTop } from './scroll-to-top';
+export { default as Gettingstarted } from './gettingstarted';
```
Comment (ajitbohra) [inline src/components/index.js:10]: Should be under pages.

File: src/components/md.css
Comment (ajitbohra) [inline src/components/md.css:1]: should be in styles > docs.css under src

**Review by ajitbohra — CHANGES_REQUESTED** [review]
Need changes

File: src/components/md.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+pre {
+    background-color: rgb(181, 181, 181);
+    padding: 20px;
+}
+
+.main {
+    margin-left: 150px;
+    margin-right: 150px;
+    border: 1px solid black;
```
Comment (ajitbohra) [inline src/components/md.css:9]: remove border here

**Review by ajitbohra — APPROVED** [review]

File: src/components/md.css
Comment (deepakgithub11) [reply src/components/md.css:1]: do we have ti make new folder for al css???

File: src/Doc/doc.css
Comment (ajitbohra) [inline src/Doc/doc.css:1]: the folder name should be a small case `doc` `doc.css` should be under styles folder

File: src/Doc/getting-started.md
Comment (ajitbohra) [inline src/Doc/getting-started.md:1]: folder name should be small case `doc`

**Review by ajitbohra — CHANGES_REQUESTED** [review]

File: src/components/banner.js [hunk 4]
```diff
@@ -48,6 +53,15 @@ function Banner() {
                         >
                             Opensource
                         </Button>
+                        <Link to="gettingstarted">
+                        <Button
+                            variant="secondary"
```
Comment (ajitbohra) [inline src/components/banner.js:58]: remove variant as it adds blue border to button

**Review by ajitbohra — CHANGES_REQUESTED** [review]

File: src/Doc/getting-started.md
Comment (ajitbohra) [inline src/Doc/getting-started.md:1]: folder name should be `doc` all small case

File: src/components/banner.js [hunk 5]
```diff
@@ -11,6 +11,11 @@ import {
     __experimentalHeading as Heading,
 } from "@wordpress/components";
 
+/**
+ * WordPress dependencies.
```
Comment (ajitbohra) [inline src/components/banner.js:15]:
```suggestion
 * External dependencies.
```

**Review by ajitbohra — CHANGES_REQUESTED** [review]

**Comment by deepakgithub11** [comment]
added build folder in .gitignore in futher PR build will be ignored

**Comment by deepakgithub11** [comment]
added build folder in .gitignore in further PR build will be ignored

**Review by ajitbohra — APPROVED** [review]
👍

**Pull request merged**